pkgs/by-name/gi/gitlab-runner/package.nix

```diff
@@ -1,33 +1,33 @@
-{ lib, buildGoModule, fetchFromGitLab, bash }:
+{
+  lib,
+  buildGoModule,
+  fetchFromGitLab,
+  bash,
+}:
 
 let
-  version = "17.2.0";
+  version = "17.9.1";
 in
 buildGoModule rec {
   inherit version;
   pname = "gitlab-runner";
 
-  commonPackagePath = "gitlab.com/gitlab-org/gitlab-runner/common";
-  ldflags = [
-    "-X ${commonPackagePath}.NAME=gitlab-runner"
-    "-X ${commonPackagePath}.VERSION=${version}"
-    "-X ${commonPackagePath}.REVISION=v${version}"
-  ];
-
   # For patchShebangs
   buildInputs = [ bash ];
 
-  vendorHash = "sha256-1MwHss76apA9KoFhEU6lYiUACrPMGYzjhds6nTyNuJI=";
+  vendorHash = "sha256-t/FVaDga2ogyqgVdJuBMSyls3rricfqIy5bFSH4snk4=";
 
   src = fetchFromGitLab {
     owner = "gitlab-org";
     repo = "gitlab-runner";
     rev = "v${version}";
-    hash = "sha256-a2Igy4DS3fYTvPW1vvDrH/DjMQ4lG9cm/P3mFr+y9s4=";
+    hash = "sha256-FGE/CVnf73u3w8Czb+NzS2qXf5OQPFI8ksTf0pKV7UY=";
   };
 
   patches = [
-    ./fix-shell-path.patch
+    ./disable-kubernetes-test-on-darwin.patch
+    ./disable-unix-socket-tests-on-darwin.patch
+    ./disable-x509-certificate-check.patch
```

Comment:
Usually a comment is added for each patch. Even if these filenames are quite explicit, it would be nice to know if these patch should be removed at the next bump.